File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/CharacterDetailVO.kt
package com.sdkwork.app

data class CharacterDetailVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val characterId: String? = null,
    val name: String? = null,
    val type: String? = null,
    val description: String? = null,
    val avatar: ImageMediaResource? = null,
    val threeViewImage: ImageMediaResource? = null,
    val gridShotsImage: ImageMediaResource? = null,
    val avatarVideo: VideoMediaResource? = null,
    val status: String? = null,
    val agentId: Int? = null,
    val personality: String? = null,
    val background: String? = null,
    val interactionSettings: String? = null,
    val version: String? = null,
    val isPublic: Boolean? = null,
    val usageCount: Int? = null,
    val likeCount: Int? = null,
    val lastUsedAt: String? = null
)

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/character_detail_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class CharacterDetailVO:
    """角色详情响应"""
    created_at: str = None
    updated_at: str = None
    character_id: str = None
    name: str = None
    type: str = None
    description: str = None
    avatar: ImageMediaResource = None
    three_view_image: ImageMediaResource = None
    grid_shots_image: ImageMediaResource = None
    avatar_video: VideoMediaResource = None
    status: str = None
    agent_id: int = None
    personality: str = None
    background: str = None
    interaction_settings: str = None
    version: str = None
    is_public: bool = None
    usage_count: int = None
    like_count: int = None
    last_used_at: str = None
